coalib/bears/Bear.py

```diff
@@ -74,6 +74,17 @@ class Bear(Printer, LogPrinter):
     ...     REQUIREMENTS = PythonRequirement.multiple(
     ...         ('colorama', '0.1'), 'coala_decorators')
 
+    To specify additional attributes to your bear, use the following:
+
+    >>> class SomeBear(Bear):
+    ...     AUTHOR = "Jon Snow"
+    ...     AUTHOR_EMAIL = "jon_snow@gmail.com"
+    ...     MAINTAINERS = ("Catelyn Stark",)
+    ...     MAINTAINERS_EMAILS = ("catelyn_stark@gmail.com",)
+    ...     INCLUDE_LOCAL_FILES = {'bears.java': ['checkstyle.jar',
+    ...                                 'google_checks.xml']}
```

Comment:
this needs documentation, why is it this syntax and not simply paths to files? Make it simpler please :)